scripts/build.sh

```diff
@@ -4,59 +4,104 @@
 
 echo "Usage: "
 echo "      ${0} clean: clean build files"
-echo "      ${0}      : mkdir build and cmake to it"
-
-cmd=$1
-if [[ "$cmd" == "clean" ]]; then
-    if [ -d build ]; then
-        cd build
-        ninja clean
-        rm CMakeCache.txt
-        rm -rf CMakeFiles
-        rm -rf cmake_install.cmake
-        rm -rf Makefile
-        rm -rf compile_commands.json
-        find . -name \CMakeCache.txt -type f -delete
-        rm build.ninja
-        rm axie_tflite
+echo "      ${0} x86  : build for x86 (in build_x86)"
+echo "      ${0} gpu  : build for arm64 Android (in build_android)"
+echo "      ${0}      : build for Android with QNN (in build_android)"
+
+arg=$1
+if [[ "$arg" == "clean" ]]; then
+    if [ -d "build_android" ]; then
+        cd build_android
+    else
+     	cd build_x86
     fi
```

Comment:
what about `build` directory? (for default target)

Reply:
ah.  all android goes under build_android, sgtm

Reply:
so 'build' is not used here any more.. it's for both QNN and GPU cases which are 'build_android' at the end. 
Do you want to separate the build folder for these ?